wgsl/index.bs

```diff
@@ -2805,8 +2825,13 @@ In particular, the token does not [=resolve=] to any declared object.
 
 Note: The token `handle` is reserved: it is never used in a WGSL program.
 
+Note: Each address space may have different performance characteristics.
+
 ### Memory Layout ### {#memory-layouts}
 
+All [=address spaces=] in WGSL share the same memory layout.
```

Comment:
Types have layouts, not address spaces...?
Do you mean the layout of a value is the same no matter what address space it may be stored  in, and that layout is completely determined by the value's type.

Reply:
Right, I'll try to clarify. We lean on the layout for sizing workgroup memory for example so it shouldn't be limited to storage and uniform.